src/Transport.ts

```diff
@@ -562,15 +565,19 @@ export class Transport extends EnhancedEventEmitter
 		// Set MID.
 		if (!pipe)
 		{
-			rtpParameters.mid = `${this._nextMidForConsumers++}`;
-
-			// We use up to 8 bytes for MID (string).
-			if (this._nextMidForConsumers === 100000000)
+			rtpParameters.mid = preferredMid;
+			if (!rtpParameters.mid)
```

Comment:
Little readability change.

```suggestion
			
			if (preferredMid)
			{
			  rtpParameters.mid = preferredMid;
			}
			else
```